package.json

```diff
@@ -5,7 +5,7 @@
   "main": "",
   "scripts": {
     "postinstall": "cd couch2pg && npm ci",
-    "test:e2e": "npm run test:e2e-data && npm run test:e2e-containers && mocha tests/**/*.spec.js --timeout 50000",
+    "test:e2e": "npm run test:e2e-data && npm run test:e2e-containers && mocha tests/**/*.spec.js --timeout 2000000",
```

Comment:
I remember running the CHT Sync tests locally ten days ago on the `main` branch, and the timeout error was not happening (and no other code changes were added since). The `main` branch e2e tests are failing for me, too, locally, and increasing the timeout to `2000000` doesn't solve it. Do you know why such a timeout would happen suddenly?
I fear this change will slow running the tests without solving the timeout's root cause.

Reply:
I have pushed a new commit that should solve the root cause and resets the timeout.